src/ui.rs

```diff
@@ -1270,6 +1309,7 @@ impl<'a> UiFrame<'a> {
                         imgui::StyleColor::ButtonHovered,
                         self.colors.special_button_hovered,
                     ),
+                    (imgui::StyleColor::TextDisabled, self.colors.special_button_text),
```

Comment:
Please sort this below "text" and above "button"

Reply:
Meh, not there.

Reply:
Not sure I understand. I did something though.